### ISSUE TITLE
Incorrect "Operation on an array doesn't make much sense here"

### ISSUE THREAD

**Issue by dstockto** [kalessil/phpinspectionsea#1804]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate), 3.0.12 |
| Language level | 8.1                        |

Current behaviour
--
Using the spaceship <=> operator to sort objects by multiple fields using arrays, the plugin is reporting "Operation on an array doesn't make much sense here". 

Example:

```
function compare(Person $a, Person $b): int 
{
    return [$b->getName(), $a->getAge()] <=> [$a->getName(), $b->getAge()];
}

The intent of the function is to provide a comparison to sort descending by name and if name matches then sort ascending by age.

Expected behaviour
--

No error.

Environment details

MacOS Big Sur, PHP 8.1
---

Information from Help - About dialog

PhpStorm 2021.3.1
Build #PS-213.6461.83, built on December 29, 2021
Runtime version: 11.0.13+7-b1751.21 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 11.6.2
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 16
Registry:
    run.processes.with.pty=TRUE

**Comment by jdreesen** [comment]
Stumbled upon this, too. 

This is clearly a false positive, as the spaceship operator allows comparing arrays (as can be seen in [the RFC](https://wiki.php.net/rfc/combined-comparison-operator)).

**Comment by ea-inspections-team** [comment]
Thank you for reporting @dstockto! The false-positive has been addressed.
